### PR TITLE
added models

### PR DESCRIPTION
**Pull request by alandchang-smoothstack**
Added models, modified index.js to require models within a directory

### CODE REVIEW

File: models/Branch.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    name: {
+        type: String,
+        required: true
+    },
+    address: {
+        type: String,
+        required: true
+    },
+    copies: [{ type: Schema.Types.ObjectId, ref: 'Copy', }],
+    borrowers: [{ type: Schema.Types.ObjectId, ref: 'Borrower', }]
```
Comment (AlexChavez235) [inline models/Branch.js]: Should be loans not borrowers.

File: models/Copy.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var copySchema = new Schema({
+    book: {
```
Comment (AlexChavez235) [inline models/Copy.js:5]: should also have a branch

File: models/Loan.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var loanSchema = new Schema({
+    book: {
+        type: Schema.Types.ObjectId, ref: 'Book',
+        required: true
```
Comment (AlexChavez235) [inline models/Loan.js:7]: should have a branch

**Review by AlexChavez235 — CHANGES_REQUESTED** [review]
Make sure to export the models too.

**Review by AlexChavez235 — APPROVED** [review]

**Review by AveryGC — APPROVED** [review]
Looks good to me

**Pull request merged**